index.js

```diff
@@ -1,5 +1,17 @@
 function locate(array, target) {
   // do work here
+  let hash = {};
+  for (let i = 0; i < array.length; i++) {
+    for (let j = 0; j < array[i].length; j++) {
+      hash[array[i][j]] = i;
+    }
+  }
```

Comment:
Although this works for this exercise, can you figure out a way to handle any depth?